include/engine/routing_algorithms/routing_base.hpp

```diff
@@ -484,6 +484,62 @@ template <class DataFacadeT, class Derived> class BasicRoutingInterface
         unpacked_path.emplace_back(t);
     }
 
+    void UnpackEdgeToEdges(const NodeID s, const NodeID t, std::vector<EdgeData> &unpacked_path) const
```

Comment:
I'd love to see us start adding docs to functions.  Good examples happening in `node-osrm` here:

https://github.com/Project-OSRM/node-osrm/blob/api-documentation/src/node_osrm.cpp#L73-L90

OSRM is woefully commented, let's fix it :-)